Directory.Build.targets

```diff
@@ -6,18 +6,6 @@
     <NETCoreAppMaximumVersion>5.0</NETCoreAppMaximumVersion>
   </PropertyGroup>
 
-  <ItemGroup>
-    <!-- Workaround while there is no 5.0 SDK available, copy known apphost/framework reference info from 3.0 -->
-    <KnownAppHostPack 
-      Include="@(KnownAppHostPack->WithMetadataValue('TargetFramework', 'netcoreapp3.0'))"
-      TargetFramework="netcoreapp5.0"
-      />
-    <KnownFrameworkReference
-      Include="@(KnownFrameworkReference->WithMetadataValue('TargetFramework', 'netcoreapp3.0'))"
-      TargetFramework="netcoreapp5.0"
-      />
-  </ItemGroup>
```

Comment:
Instead make this logic conditional on lack of `%(TargetFramework) == $(DefaultNetCoreTargetFramework)` items